Lib/ufo2ft/outlineOTF.py

```diff
@@ -942,6 +953,8 @@ def setupTable_post(self):
 
     def setupOtherTables(self):
         self.setupTable_glyf()
+        if getattr(self.ufo.info, "openTypeGaspRangeRecords"):
```

Comment:
Is the getattr for robofab compability? With defcon I guess you can always call self.ufo.info.openTypeGaspRangeRecords